back/engines/commercial/id_franceconnect/app/lib/id_franceconnect/feature_specification.rb

```diff
@@ -46,5 +46,17 @@ def self.enabled_by_default
       type: 'string',
       private: true
     }
+
+    add_setting 'scope', required: true, schema: {
```

Comment:
Previously, we've been changing the scope to-and-fro multiple times
to satisfy specific customer's requirements.

In some cases, we needed to remove some fields:
https://citizenlabco.slack.com/archives/C01NLAC43T3/p1647964640898609?thread_ts=1647964476.114329&cid=C01NLAC43T3

In other cases, we needed to add some fields:
see the attachments here https://citizenlab.atlassian.net/browse/CL-3422

Possibly, they were just configuration issues (and now it'll be easier
to solve them), but it'll be also useful to have the ability to save
some additional fields.